tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -566,6 +587,28 @@ public function theUserCreatesNewUser(string $user, TableNode $table): void {
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @Given /^the user "([^"]*)" has created a new user using the Graph API with the following settings:$/
+	 *
+	 * @param string $user
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception|GuzzleException
+	 */
+	public function theUserHasCreatedANewUserUsingGraphapiWithTheFollowingSettings(string $user, TableNode $table): void {
+		$this->theUserCreatesNewUser(
+			$user,
+			$table
+		);
+		$rows = $table->getRowsHash();
+		$response = $this->featureContext->getResponse();
+
+		if ($response->getStatusCode() !== 200) {
```

Comment:
```suggestion
$this->featureContext->theHttpStatusCodeShouldBeSuccessful()
```
isn't there something already like this?

Reply:
The purpose of this `if block` is to give a proper error message, methods like `theHttpStatusCodeShouldBeSuccessful` give an error message related to the status code. IMO that won't fit here.